pkg/strategy/production.go

```diff
@@ -87,6 +88,9 @@ func newProductionStrategy(ctx context.Context, jaeger *v1.Jaeger) S {
 	if viper.GetString("platform") == v1.FlagPlatformOpenShift {
 		if q := route.NewQueryRoute(jaeger).Get(); nil != q {
 			c.routes = append(c.routes, *q)
+			if link := consolelink.Get(jaeger, q); link != nil {
```

Comment:
It seems that you are missing tests for this. Same for the other strategies.

Reply:
Added.